src/fileinfo/file_presentation/getters/iterative_getter/iterative_distribution_getter/missing_deps_plain_getter.cpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * @file src/fileinfo/file_presentation/getters/iterative_getter/iterative_distribution_getter/loader_info_plain_getter.cpp
+ * @brief Methods of MissingDepsPlainGetter class.
+ * @copyright (c) 2017 Avast Software, licensed under the MIT license
+ */
+
+#include "retdec/utils/string.h"
+#include "retdec/fileformat/utils/conversions.h"
+#include "retdec/fileformat/utils/other.h"
+#include "fileinfo/file_presentation/getters/format.h"
+#include "fileinfo/file_presentation/getters/iterative_getter/iterative_distribution_getter/missing_deps_plain_getter.h"
+
+using namespace retdec::fileformat;
+
+namespace fileinfo {
+
+namespace
+{
+
+const static std::vector<std::size_t> distributionArray = {6, 20};
+const static std::vector<std::string> headerArray = {"i", "name"};
+const static std::vector<std::string> headerDesc = {"index", "missing dependency"};
+
+}
+
+/**
+ * Constructor
+ * @param fileInfo Information about file
+ */
+MissingDepsPlainGetter::MissingDepsPlainGetter(FileInformation &fileInfo) : IterativeDistributionGetter(fileInfo)
+{
+	numberOfStructures = 1;
+	numberOfStoredRecords.push_back(fileinfo.getNumberOfMissingDeps());
+	numberOfExtraElements.push_back(0);
+	title = "Missing dependencies";
+	distribution.insert(distribution.begin(), std::begin(distributionArray), std::end(distributionArray));
+	commonHeaderElements.insert(commonHeaderElements.begin(), std::begin(headerArray), std::end(headerArray));
+	commonHeaderDesc.insert(commonHeaderDesc.begin(), std::begin(headerDesc), std::end(headerDesc));
+	loadRecords();
+}
+
+/**
+ * Destructor
+ */
+MissingDepsPlainGetter::~MissingDepsPlainGetter()
+{
+}
+
+std::size_t MissingDepsPlainGetter::getBasicInfo(std::size_t structIndex, std::vector<std::string> &desc, std::vector<std::string> &info) const
+{
+	char szNumDeps[0x40];
+
+	if(structIndex >= numberOfStructures)
+	{
+		return 0;
+	}
+
+	desc.push_back("Number of missing dependencies: ");
+	sprintf(szNumDeps, "%u", (unsigned)fileinfo.getNumberOfMissingDeps());
```

Comment:
We have conversion functions available in `utils/conversion.h`. `numToStr` can be used here.